lib/trivial.nix

```
@@ -467,6 +471,41 @@ rec {
     then v
     else k: v;
 
+  /*
+    Make a function lazy in its arguments.
```

```suggestion
    Make a function lazy in its argument.
```

I don't like how this detail matters, but it does because there's only one actual argument here, the destructuring only makes it seem like multiple arguments. And lazy in argument**s** could be interpreted as making the attribute values lazy instead of strict (but that part is always lazy)